lambda/actions/src/controllers/batch.ts

```diff
@@ -85,8 +87,21 @@ export const updatePlannedItems = async (data) => {
         (await models.event.count({ where: { eventCode: EVENTS.REQUEST_APPLY_SUCCESS, requestId: integration.id } })) >
         1;
 
-      const emailCode = isUpdate ? EMAILS.UPDATE_INTEGRATION_APPROVED : EMAILS.CREATE_INTEGRATION_APPROVED;
-      await sendTemplate(emailCode, { integration });
+      if (integration.apiServiceAccount) {
+        const teamIntegrations = await models.request.findAll({
+          where: { teamId: integration.teamId, apiServiceAccount: false, archived: false },
```

Comment:
can we add the `attributes` option to select only necessary information?

Reply:
done!